src/Feliz.Bulma.Switch/Switch.fs

```diff
@@ -0,0 +1,12 @@
+namespace Feliz.Bulma.Switch
+
+open Feliz
+open Feliz.Bulma
+
+module private ElementLiterals =
+    [<Literal>]
+    let switch = "switch"
+
+type Switch =
```

Comment:
I think it is missing a `[<Fable.Core.Erase>]` attribute here.

Reply:
Am I? I based these changes on this other one here? 

https://github.com/Dzoukr/Feliz.Bulma/blob/cd7656918b4464ecfade7344a4a3d0f4fa137f8f/src/Feliz.Bulma.Checkradio/Checkradio.fs#L9

Reply:
Then it is also an omission in Checkradio package which should be fixed.

Adding `Erase` attribute tells fable to erase the type from the output.

![image](https://user-images.githubusercontent.com/4760796/81046737-4ff87b80-8eb9-11ea-8c2b-030d30d59b20.png)

![image](https://user-images.githubusercontent.com/4760796/81046748-5850b680-8eb9-11ea-94b1-d3a942040562.png)

This is just to reduce the impact on the bundle size as all the member are inlined we don't need the type information at runtime.

Reply:
I see. I could go over both packages and look at others and create a PR for that if you'd like @Dzoukr. Lemme know.

Reply:
That would be cool, but not forcing you, I can do it later. 😉